Cabal/src/Distribution/Simple/BuildPaths.hs

```diff
@@ -58,7 +59,8 @@ import Distribution.Simple.Utils
 import Distribution.Utils.Path
 
 import Data.List (stripPrefix)
-import System.FilePath ((</>), (<.>), normalise)
+import Data.List as List
```

Comment:
maybe put `sortBy` in line 61 (where `import Data.List` lies)

Reply:
Sure thing!